app/javascript/spec/workflow-repository-form/workflow-repository-form.spec.js

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import toJson from 'enzyme-to-json';
+import fetchMock from 'fetch-mock';
+import { shallow } from 'enzyme';
+import WorkflowRepositoryForm from '../../components/workflow-repository-form';
+
+require('../helpers/set_fixtures_helper.js');
+require('../helpers/old_js_file_require_helper.js');
+require('../helpers/miqSparkle.js');
+require('../helpers/miqAjaxButton.js');
+
+jest.mock('../../helpers/miq-redirect-back', () => jest.fn());
+
+describe('Ansible Repository form component', () => {
```

Comment:
```suggestion
describe('Workflows Repository form component', () => {
```

Similar comment anywhere else in these specs.